packages/firebase_analytics/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.0.0
+
+* Android : Remove deprecated method `setMinimumSessionDuration`.
```

Comment:
Nit: Can you make this more consistent with other CHANGELOG entries, e.g. the one below it.? Make sure to indicate the removal of methods as a breaking change.